client/reader/tag-stream/main.jsx

```diff
@@ -109,6 +109,23 @@ class TagStream extends React.Component {
 			} );
 		}
 
+		if ( ! tag || tag.error ) {
+			return (
+				<React.Fragment>
```

Comment:
ooooooh fancy

Reply:
🌟

Reply:
I didn't want to use `<> </>` because i don't think prettier knew how to handle it